apps/full-site-editing/full-site-editing-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -49,6 +49,10 @@ function updateEditor() {
 			return;
 		}
 		clearInterval( awaitSettingsBar );
+
+		const isMobile = window.innerWidth < 768;
+		const isNewLaunch = ! window?.calypsoifyGutenberg?.isNewLaunch;
```

Comment:
This is setting `isNewLaunch` to negate the value passed to `calypsoifyGutenberg`. If we're trying to switch off the new launch feature, would it be better to do this in the `gutenboarding/new-launch` setting in our config files?

Reply:
This will be resolved in https://github.com/Automattic/wp-calypso/pull/44855 (Razvan explained that there was a missing change in `wpcom-block-editor` required to get this value passing through, so negating this falsy value had the same effect as if we'd passed the expected `gutenboarding/new-launch` value of `true`).